README.md

```diff
@@ -200,6 +200,12 @@ Also, next optional tools are used to build additional resources:
 - GraphViz to generate the documentation. See [the official GraphViz website](https://www.graphviz.org/download/) for more information or use [Homebrew](https://brew.sh) to install it.
 - `protobuf-c` to convert Protocol Buffer `.proto` files to C descriptor code. See [the official protobuf-c repository](https://github.com/protobuf-c/protobuf-c) for more information or use [Homebrew](https://brew.sh) to install it.
 
+### Proto files modification
+If you have modified any files with the .proto extension (such as `Tests/PLCrashLogWriterEncodingTests.proto` or `Source/PLCrashReport.proto`), then before building the project, you need to generate the source files using `protobuf-c`. To do this, you can run the following script:
```

Comment:
Maybe I am wrong, but probably you could customize this behavior and make it automatic with build rules. It's just a suggestion.

Reply:
It is a manual process on updating protobuf. It's not linked to the build.

Reply:
Previously, this process was automated until we encountered the issue described here: [GitHub Issue #317](https://github.com/microsoft/plcrashreporter/issues/317). To resolve the problem, we had to discontinue the automated proto-c file generation via Xcode, as addressed in this [pull request](https://github.com/microsoft/plcrashreporter/pull/327).